### PR TITLE
482 add configuration

### PR DESCRIPTION
**Pull request by kotbegemot**

### CODE REVIEW

File: share/golosd/config/config_witness.ini [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+# api-user =
+
+webserver-thread-pool-size = 2
+webserver-http-endpoint = 0.0.0.0:8090
+webserver-ws-endpoint = 0.0.0.0:8091
+rpc-endpoint = 0.0.0.0:2001
+
+shared-file-size = 128G
+
+plugin = chain p2p json_rpc webserver network_broadcast_api witness
```
Comment (afalaleev) [inline share/golosd/config/config_witness.ini]: `json_rpc` and `webserver` are used for commutations with clients, but there is no API plugins. May be add database_api for basic API functions without additional indexes? What do you think?

File: share/golosd/config/config_witness.ini [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+# edit-script =
+
+# RPC endpoint of a trusted validating node (required)
+# trusted-node =
+
+# Set the maximum size of cached feed for an account
+follow-max-feed-size = 500
+
+# Track market history by grouping orders into buckets of equal size measured in seconds specified as a JSON array of numbers
+bucket-size = [15,60,300,3600,86400]
```
Comment (afalaleev) [inline share/golosd/config/config_witness.ini]: This option is used by `market_history`. Is it really needed here?

File: share/golosd/config/config_witness.ini [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+# filter-posting-ops =
+
+# Database edits to apply on startup (may specify multiple times)
+# edit-script =
+
+# RPC endpoint of a trusted validating node (required)
+# trusted-node =
+
+# Set the maximum size of cached feed for an account
+follow-max-feed-size = 500
```
Comment (afalaleev) [inline share/golosd/config/config_witness.ini]: This option is used by `follow`. Is it really needed here?

File: share/golosd/config/config_witness.ini [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+# API user specification, may be specified multiple times
+# api-user =
+
+webserver-thread-pool-size = 2
+webserver-http-endpoint = 0.0.0.0:8090
+webserver-ws-endpoint = 0.0.0.0:8091
+rpc-endpoint = 0.0.0.0:2001
+
+shared-file-size = 128G
```
Comment (afalaleev) [inline share/golosd/config/config_witness.ini]: For small `witness` node, 32G is enough.

File: share/golosd/config/config_witness.ini [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+# trusted-node =
+
+# Set the maximum size of cached feed for an account
+follow-max-feed-size = 500
+
+# Track market history by grouping orders into buckets of equal size measured in seconds specified as a JSON array of numbers
+bucket-size = [15,60,300,3600,86400]
+
+# How far back in time to track history for each bucket size, measured in the number of buckets (default: 5760)
+history-per-size = 5760
```
Comment (afalaleev) [inline share/golosd/config/config_witness.ini]: This option is used by `market_history`. Is it really needed here?

File: share/golosd/config/config_stock_exchange.ini [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+# filter-posting-ops =
+
+# Database edits to apply on startup (may specify multiple times)
+# edit-script =
+
+# RPC endpoint of a trusted validating node (required)
+# trusted-node =
+
+# Set the maximum size of cached feed for an account
+follow-max-feed-size = 500
```
Comment (afalaleev) [inline share/golosd/config/config_stock_exchange.ini]: This option is used by `follow`. Is it really needed here?

File: share/golosd/config/config_stock_exchange.ini [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+webserver-ws-endpoint = 0.0.0.0:8091
+rpc-endpoint = 0.0.0.0:2001
+
+shared-file-size = 128G
+
+plugin = chain p2p json_rpc webserver network_broadcast_api witness database_api block_info raw_block account_history
+
+
+# JSON list of [nblocks,nseconds] pairs, see documentation/bcd-trigger.md
+bcd-trigger = [[0,10],[85,300]]
```
Comment (afalaleev) [inline share/golosd/config/config_stock_exchange.ini]: For which plugin this option is required?

File: share/golosd/config/config_stock_exchange.ini [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+# edit-script =
+
+# RPC endpoint of a trusted validating node (required)
+# trusted-node =
+
+# Set the maximum size of cached feed for an account
+follow-max-feed-size = 500
+
+# Track market history by grouping orders into buckets of equal size measured in seconds specified as a JSON array of numbers
+bucket-size = [15,60,300,3600,86400]
```
Comment (afalaleev) [inline share/golosd/config/config_stock_exchange.ini]: This option is used by `market_history`. Is it really needed here?

File: share/golosd/config/config_stock_exchange.ini [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+
+# How far back in time to track history for each bucket size, measured in the number of buckets (default: 5760)
+history-per-size = 5760
```
Comment (afalaleev) [inline share/golosd/config/config_stock_exchange.ini]: This option is used by `market_history`. Is it really needed here?

File: share/golosd/config/config_stock_exchange.ini [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+# filter-posting-ops =
+
+# Database edits to apply on startup (may specify multiple times)
+# edit-script =
+
+# RPC endpoint of a trusted validating node (required)
+# trusted-node =
+
+# Track market history by grouping orders into buckets of equal size measured in seconds specified as a JSON array of numbers
+bucket-size = [15,60,300,3600,86400]
```
Comment (afalaleev) [inline share/golosd/config/config_stock_exchange.ini]: This option is used by `market_history`. Is it really needed here?

**Comment by bitfag** [comment]
For witness nodes, you can also create a LOW_MEM docker image.

**Pull request merged**